crates/backend/src/ast.rs

```diff
@@ -802,116 +807,94 @@ impl BindgenAttrs {
     fn module(&self) -> Option<&str> {
         self.attrs
             .iter()
-            .filter_map(|a| {
-                match a {
-                    BindgenAttr::Module(s) => Some(&s[..]),
-                    _ => None,
-                }
+            .filter_map(|a| match a {
+                BindgenAttr::Module(s) => Some(&s[..]),
+                _ => None,
             })
             .next()
     }
 
     fn version(&self) -> Option<&str> {
         self.attrs
             .iter()
-            .filter_map(|a| {
-                match a {
-                    BindgenAttr::Version(s) => Some(&s[..]),
-                    _ => None,
-                }
+            .filter_map(|a| match a {
+                BindgenAttr::Version(s) => Some(&s[..]),
+                _ => None,
             })
             .next()
     }
 
     pub fn catch(&self) -> bool {
-        self.attrs.iter().any(|a| {
-            match a {
-                BindgenAttr::Catch => true,
-                _ => false,
-            }
+        self.attrs.iter().any(|a| match a {
+            BindgenAttr::Catch => true,
+            _ => false,
         })
     }
 
     fn constructor(&self) -> bool {
-        self.attrs.iter().any(|a| {
-            match a {
-                BindgenAttr::Constructor => true,
-                _ => false,
-            }
+        self.attrs.iter().any(|a| match a {
```

Comment:
Hm is there a way we can disable this in rustfmt? This seems like a bug?

Reply:
I can undo it certainly. Not sure there is a way to tell rustfmt not to do it in the first place, and I'm not really a fan of deviating from rustfmt's behavior anyways. /me shrugs

Reply:
Nah I know what you mean. I specifically really dislike this style of formatting because when you read the body you have no idea it's in ` match` unless you go back up and read it (and the `match` is hidden with the closure), so I'm sort of hoping this is a bug in rustfmt that'll get fixed at some point...